homework/shelkovich_ilya/functional_interfaces/src/main/java/com/tutrit/java/quickstart/bean/Person.java

```diff
@@ -0,0 +1,52 @@
+package com.tutrit.java.quickstart.bean;
+
+import java.util.Objects;
+
+public class Person {
+
+    private String name;
+    private int age;
+
+    public Person(String name, int age) {
+        this.name = name;
+        this.age = age;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public Person setName(String name) {
+        this.name = name;
+        return this;
+    }
+
+    public int getAge() {
+        return age;
+    }
+
+    public Person setAge(int age) {
+        this.age = age;
+        return this;
+    }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
+        if (o == null || getClass() != o.getClass()) return false;
+        Person person = (Person) o;
+        return age == person.age && Objects.equals(name, person.name);
+    }
+
+    @Override
+    public int hashCode() {
+        return Objects.hash(name, age);
+    }
+
+    @Override
+    public String toString() {
+        return "Person{" + "name='" + name + '\'' +
+                ", age=" + age +
+                '}';
+    }
+}
```

Comment:
Add a blank line to the end of the file
everywhere.

Reply:
kk